source/content/drupal-9.md

```diff
@@ -76,6 +76,110 @@ $node = Node::load(1);
 
 Since most of these changes are relatively minor, there are a number of [deprecation checking and correction tools](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9/deprecation-checking-and-correction-tools) available.
 
+## Convert an existing Drupal 8 site to Drupal 9
+
+Steps to convert and migrate your Drupal 8 site to a new Drupal 9 instance.
+
+Prerequisites:
+- Install Terminus, Git, and Composer (see "[Before You Begin](drupal-9#before-you-begin)" above)
+- Install the [Terminus Site Clone](https://github.com/pantheon-systems/terminus-site-clone-plugin) plugin
+- Set environment variables with your Drupal 8 and Drupal 9 site names:
+
+```bash
+export D8_SITE=best-drupal8-site-ever
+export D9_SITE=best-drupal9-site-ever
+```
+
+1. Create a new Drupal 9 site using the link in the [Create a New Drupal 9 Site](drupal-9#create-a-new-drupal-9-site-with-integrated-composer) section above.
+
+2. Use Git to clone your new Drupal 9 site codebase to your local machine.
+
+3. On the Drupal 9 site, add your Drupal 8 site repo as remote. You can retrieve the D8 site's Git URL via the Dashboard, or with Terminus:
+
+```bash
+terminus connection:info $D8_SITE.dev --field=git_url
+```
+
+Add this as a remote on the D9 site:
+
+```bash
+git remote add original ssh://codeserver.dev.xxxx@codeserver.dev.xxxx.drush.in:2222/~/repository.git
```

Comment:
Re-reading these steps a few days later, I am wondering if perhaps `original` was a poor choice for the upstream name. It sounds a lot like `origin`, which could be confusing.

Reply:
agreed - and could pose an additional problem for someone who uses tab completes and maybe isn't paying attention
renamed to `existing-8`